### DISCUSSION TITLE
registry_package doesn't trigger on Docker image push

### DISCUSSION THREAD

**Discussion by jayarjo** [community/community#25123] [Actions]
<p>I have an action that starts with:</p>
<pre><code>name: Deploy updated image to the cloud

on:
  registry_package:
    types: [published]
</code></pre>
<p>but despite the Docker image being pushed onto the registry workflow nevers triggers. I’m missing anything, or there some additional steps to undertake for this to work? I fail to notice anything such in the docs <a href="https://help.github.com/en/actions/reference/events-that-trigger-workflows#registry-package-event-registry_package" rel="nofollow noopener">here</a>.</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/jayarjo">@jayarjo</a> ,</p>
<p>Did you use personal access token as a secret to push the docker image? If you use ‘GITHUB_TOKEN’ to push the docker image, ‘registry_package’ event will not be triggered.</p>
<p>And please use below format instead,</p>
<pre><code>on:
  registry_package:
    action:
      - published
      - updated
</code></pre>
<p>Checked on my side, It is successful.</p>
<p><img src="https://user-images.githubusercontent.com/52772761/181089035-b78e62db-8b32-4597-8630-8c628cbfd5af.png" alt="Packagetrigger.png" data-base62-sha1="5eAXb7nmAu3u1A6GQWT0AkBEGcj" width="662" height="151"></p>
<p>If it doesn’t work for you, please share your repo/sample repo for further investigation.</p>
<p>Thanks.</p>

**Comment by jayarjo** [comment]
<p>Initially I was trying with GITHUB_TOKEN indeed and then stumbled on a post that mentioned requirement for the personal token (I think documentation for registry_package should also mention this requirement), so I tried that as well - it didn’t work. I definitely tried the syntax in your screenshot as well. In fact it was my initial syntax before I switched to the one mentioned in docs. Nothing worked.</p>
<p>But now it works. The syntax from your screenshot + personal token. Not sure. Thanks anyway!</p>

**Comment by vasylherman** [comment]
<p>Hello, can you please check if it triggers on “- updated” only?</p>
<pre><code>on:
registry\_package:
action:  
 - updated 
</code></pre>
